uiServices/src/testCaseServices.go

```diff
@@ -294,24 +307,27 @@ func deleteTestCase(rw http.ResponseWriter, req *http.Request) {
 }
 func deleteAllTestCases(rw http.ResponseWriter, req *http.Request) {
 	testCasePathDir := getTestCaseHeader(req)
-	logrus.Println("testCasePathDir", testCasePathDir)
+
 	if err := IsHeaderValid(testCasePathDir); err != nil {
+		logrus.Error("No heared path found", err)
 		rw.WriteHeader(http.StatusBadRequest)
 		return
 	}
 
 	files, err := ioutil.ReadDir(testCasePathDir)
 	if err != nil {
-		logrus.Error(err)
+		logrus.Error("Error reading the directory ", err)
+		rw.WriteHeader(http.StatusInternalServerError)
+		return
 	}
 
 	for _, file := range files {
 		filepath := fmt.Sprintf("%s%s", testCasePathDir, file.Name())
 
 		err := os.Remove(filepath)
 		if err != nil {
-			logrus.Errorf("Error deleting the files from directory: %s", err)
-			rw.WriteHeader(http.StatusNotFound)
+			logrus.Error("Error removing the files from directory", err)
+			rw.WriteHeader(http.StatusNoContent)
 			return
```

Comment:
failure to remove a file should fail silently or return an internalserver error.

Reply:
@jenny-flynn   you suggested yesterday that http.StatusNoContent would be better

Reply:
yep sorry i misread the err != nil condition. internalservererror here, and no content if err == nil e.g. the last line of the function. sorry!